chronicle/src/main/java/net/openhft/chronicle/tcp/SinkTcp.java

```diff
@@ -65,7 +84,8 @@ public boolean open(boolean blocking) throws IOException {
                 socketChannel.socket().setSendBufferSize(this.builder.sendBufferSize());
             }
 
-            super.setSocketChannel(socketChannel);
+            this.socketChannel = socketChannel;
+            this.tcpEventHandler = new TcpEventHandler(socketChannel, builder.tcpPipeline(sinkTcpHandler), sessionDetailsProvider, builder.sendBufferSize(), builder.receiveBufferSize());
```

Comment:
The class TcpEventHandler sets the socket as [non-blocking](https://github.com/ryanlea/Chronicle-Queue/blob/tcp-pipeline/chronicle/src/main/java/net/openhft/chronicle/network/TcpEventHandler.java#L31) by default but the sink is supposed to be configured also as non blocking, how is it implemented here ?